kairon/shared/models.py

```diff
@@ -23,6 +23,7 @@ class StoryStepType(str, Enum):
     hubspot_forms_action = "HUBSPOT_FORMS_ACTION"
     razorpay_action = "RAZORPAY_ACTION"
     two_stage_fallback_action = "TWO_STAGE_FALLBACK_ACTION"
+    pyscript_action = "PYSCRIPT_ACTION"
```

Comment:
story step type has been added but list stories changes are not done! please check get_stories() and do the needful